app/lib/main/plugin_initializations.dart

```diff
@@ -101,6 +102,25 @@ class PluginInitializations {
   }
 }
 
+Future<void> _fetchRemoteConfig(RemoteConfiguration remoteConfiguration) async {
+  try {
+    await remoteConfiguration.fetch();
+  } catch (e) {
+    if (isIntegrationTest && PlatformCheck.isAndroid) {
+      if ('$e'.contains(
+          '[firebase_remote_config/internal] internal remote config fetch error')) {
+        print(
+            "Catched '$e'. Ignoring because we're running an integration test.");
```

Comment:
```suggestion
        log(
            "Catched '$e'. Ignoring because we're running an integration test.");
```

Shouldn't it be this?